plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizardUtil.java

```diff
@@ -0,0 +1,303 @@
+        StringBuilder sb = new StringBuilder();
+        URL url = new URL(HumantaskEditorConstants.DUMMY_WSDL_LOCATION);
+        try (InputStream inputStream = url.openConnection().getInputStream();
+                BufferedReader in = new BufferedReader(new InputStreamReader(inputStream))) {
+            String inputLine;
+            while ((inputLine = in.readLine()) != null) {
+                sb.append(inputLine).append("\n");
+            }
+        } catch (IOException e) {
+            IStatus editorStatus = new Status(IStatus.ERROR, Activator.PLUGIN_ID, e.getMessage());
+            ErrorDialog.openError(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell(),
+                    HumantaskEditorConstants.ERROR_MESSAGE, "Error reading from project", editorStatus);
+        }
+        return sb.toString();
+    }
+
+    /**
+     * Read dummy org schema file which is needed to initialize a new ht file
+     *
+     * @throws IOException
+     */
+    public String readDummyOrgSchema() throws IOException {
+        StringBuilder sb = new StringBuilder();
+        URL url = new URL(HumantaskEditorConstants.DUMMY_ORG_SCHEMA_LOCATION);
+        try (InputStream inputStream = url.openConnection().getInputStream();
+                BufferedReader in = new BufferedReader(new InputStreamReader(inputStream))) {
+            String inputLine;
+            while ((inputLine = in.readLine()) != null) {
+                sb.append(inputLine).append("\n");
+            }
+        } catch (IOException e) {
+            IStatus editorStatus = new Status(IStatus.ERROR, Activator.PLUGIN_ID, e.getMessage());
+            ErrorDialog.openError(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell(),
+                    HumantaskEditorConstants.ERROR_MESSAGE, "Error reading from project", editorStatus);
+        }
+        return sb.toString();
+    }
+
+    /**
+     * Read dummy pom file which is needed to initialize a new ht file
+     *
+     * @throws IOException
+     */
+    public String readDummyPomSchema() throws IOException {
+        StringBuilder sb = new StringBuilder();
+        URL url = new URL(HumantaskEditorConstants.DUMMY_POM_SCHEMA_LOCATION);
+        try (InputStream inputStream = url.openConnection().getInputStream();
+                BufferedReader in = new BufferedReader(new InputStreamReader(inputStream))) {
+            String inputLine;
+            while ((inputLine = in.readLine()) != null) {
+                sb.append(inputLine).append("\n");
+            }
+        } catch (IOException e) {
+            IStatus editorStatus = new Status(IStatus.ERROR, Activator.PLUGIN_ID, e.getMessage());
+            ErrorDialog.openError(PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell(),
+                    HumantaskEditorConstants.ERROR_MESSAGE, "Error reading from project", editorStatus);
+        }
+        return sb.toString();
+    }
+
+    public String readDummyHtConfig() throws IOException {
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/bebac336cdb59f71f90a982a4abb594e3095fed4